src/lib/CRSF/CRSF.cpp

```diff
@@ -326,10 +383,10 @@ void ICACHE_RAM_ATTR CRSF::sendSyncPacketToTX(void *pvParameters) // in values i
 #endif
                         if (BadPktsCount >= GoodPktsCount)
                         {
```

Comment:
Oops, picked up this change by accident doing the review changes. The bad:good count is printed unconditionally a few lines below, so these prints are duplicates.